client/me/purchases/titles.js

```diff
@@ -19,3 +19,50 @@ export default {
 	paymentMethods: i18n.translate( 'Payment Methods' ),
 	pendingPayments: i18n.translate( 'Pending Payments' ),
 };
+
+/**
+ * Define properties with translatable strings getters.
+ */
+Object.defineProperties( titles, {
+	addCreditCard: {
+		get: () => i18n.translate( 'Add Credit Card' ),
+	},
+	addPaymentMethod: {
+		get: () => i18n.translate( 'Add Payment Method' ),
+	},
+	cancelPurchase: {
+		get: () => i18n.translate( 'Cancel Purchase' ),
+	},
+	confirmCancelDomain: {
+		get: () => i18n.translate( 'Cancel Domain' ),
+	},
+	editCardDetails: {
+		get: () => i18n.translate( 'Change Credit Card' ),
+	},
+	changePaymentMethod: {
+		get: () => i18n.translate( 'Change Payment Method' ),
+	},
+	addCardDetails: {
+		get: () => i18n.translate( 'Add Credit Card' ),
+	},
+	managePurchase: {
+		get: () => i18n.translate( 'Purchase Settings' ),
```

Comment:
This appears to be untranslated when going to details from 'Active Upgrades' section.